assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss

```diff
@@ -147,4 +147,57 @@
 			margin-bottom: 0;
 		}
 	}
+
+	.googlesitekit-ad-blocking-recovery__step {
+
+		.googlesitekit-stepper__step-content {
+
+			> p {
+				color: $c-surfaces-on-surface-variant;
+				font-size: $fs-body-lg;
+				letter-spacing: $ls-s;
+			}
+
+			> p:first-child {
+				margin-top: $grid-gap-phone / 2;
+			}
+		}
+	}
+
+	.googlesitekit-ad-blocking-recovery__step-place-tags {
+
+		p.googlesitekit-ad-blocking-recovery__error-protection-tag-info {
+			font-size: $fs-body-md;
+			line-height: $lh-body-md;
+			margin: 0 0 20px 34px; // Aligns the paragraph with the checkbox.
+		}
+	}
+
+	.googlesitekit-ad-blocking-recovery__step-create-message {
+
+		.googlesitekit-ad-blocking-recovery__create-message-footer-actions {
+			align-items: center;
+			display: flex;
+			flex-wrap: wrap;
+			gap: $grid-gap-desktop + 2px;
```

Comment:
Why the extra 2px? 🤔

Reply:
According to the Figma designs, the buttons have a gap of 26px in between:

![image](https://github.com/google/site-kit-wp/assets/20284937/cb1a11b1-5b65-4a48-9970-770a3c028b99)

Since `$grid-gap-desktop` is set to 24px, I added 2px to match the designs. Should I remove it?

Reply:
Ah, I see now. So the secondary choice there is actually a button in the design, but we use a link. The two have a 10px gap between.
<img width="812" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/e5b1bc3a-bb40-412f-ac52-8dca75b2b4e2">

We probably don't have a great solution for this currently I don't think. Ideally we'd have maybe an `ActionsRow` component in the future to have a consistent layout for this kind of thing but for now, using a `26` would be okay, I'd just add a comment as to where the value is coming from. This is something we should address throughout, so we don't need to get overly hung up on it here 👍

Reply:
Ah, I see, that makes sense. I have added a comment to explain where the extra 2px comes from. Thank you!